web/src/app/browser/tsconfig.json

```diff
@@ -2,13 +2,9 @@
   "extends": "../../tsconfig.dom.json",
 
   "compilerOptions": {
-    "baseUrl": "./",
```

Comment:
Per [Typescript documentation](https://www.typescriptlang.org/docs/handbook/modules/reference.html#baseurl), this is not generally required any more.